packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -880,23 +880,26 @@ class SlideshowView extends GalleryComponent {
     }
 
     const { customNavArrowsRenderer } = this.props;
-    let arrowRenderer, navArrowsContainerSize;
+    let arrowRenderer, navArrowsContainerWidth, navArrowsContainerHeight;
```

Comment:
I prefer to avoid mutations in the code. It makes it much more difficult to track what is changing and by whom. You can do something llike this:
```
{navArrowsContainerWidth, navArrowsContainerHeight} = getHeightAndWidth(customNavArrowsRenderer)
```
The `getHeightAndWidth` function is a pure function that can be outside of the class (can still be in the same file if you want)